UI/src/sass/mobile-views.scss

```diff
@@ -0,0 +1,7 @@
+﻿@media(max-width: 40.0625em) {
```

Comment:
Replace `@media(max-width: 40.0625em)` with `@include govuk-media-query($until: tablet)` and `padding-left: 25px` with `padding-left: govuk-spacing(5);` to use the [GOV.UK frontend Sass API](https://frontend.design-system.service.gov.uk/sass-api-reference/). I'd also bookmark that reference as it's useful for the future

Reply:
Changed